docs/data/xrootd/xrootd-authorization.md

```diff
@@ -184,14 +185,24 @@ For example,
 ```
 will map FQANs starting with `/osg/` to the user `osg01`.
 
+To enable using VOMS mapfiles in the first place, add the following line to your XRootD configuration:
+```
+voms.mapfile /etc/grid-security/voms-mapfile
+```
+replacing `/etc/grid-security/voms-mapfile` with the actual location of your mapfile, if it is different.
```

Comment:
```suggestion
To enable using VOMS mapfiles in the first place, add the following line to your XRootD configuration:
```
voms.mapfile /etc/grid-security/voms-mapfile
```
replacing `/etc/grid-security/voms-mapfile` with the actual location of your mapfile, if it is different.
```

I had a comment saying that we do this in `osg-xrootd` but we don't 🤦🏽

Reply:
yep, we don't.

Reply:
we should =/